character-controller.js

```diff
@@ -977,16 +977,28 @@ class UninterpolatedPlayer extends StatePlayer {
       emote: new BiActionInterpolant(() => this.hasAction('emote'), 0, crouchMaxTime),
       movements: new InfiniteActionInterpolant(() => {
         const ioManager = metaversefile.useIoManager();

```

Comment:
We could probably move the useIoManager call up to the top of the function and then re-use it

Reply:
I tried to move it up and reuse it, but when doing so, `metaversefile` somehow is only an empty `EventTarget` object and `useIoManager()` isn't defined. So I'll keep it as it is for now.